app/views/your-psp/_worldpay.njk

```diff
@@ -11,7 +14,7 @@
           text: "Merchant code"
         },
         value: {
-          text: credential.credentials.merchant_id if isAccountCredentialsConfigured else "Not configured",
+          text: credential.credentials.one_off_customer_initiated.merchant_code if isAccountCredentialsConfigured else "Not configured",
```

Comment:
This is relying on the fact that https://github.com/alphagov/pay-connector/commit/07a376be488195a3b2b673a72043018bc70595ee synthesises a `one_off_customer_initiated` object for credentials stored in the legacy way, right?